testing/src/scenario/runtime.py

```diff
@@ -151,6 +153,7 @@ def __enter__(self):
         pass
 
     def emit(self):
+        """Emit the event."""
         self._has_emitted = True
```

Comment:
No, it's in a private class, so users don't interact with it at all. It's not a great docstring, but it's also scheduled for removal (we'll use the new(ish) _Manager class instead of this stuff) so I didn't take much care.

Basically, for the testing case we don't need to do anything here, other than record that happened.

I'll make it better.